packages/web-react/src/components/UNSTABLE_PartnerLogo/__tests__/usePartnerLogoStyleProps.test.ts

```diff
@@ -10,18 +10,27 @@ describe('usePartnerLogoStyleProps', () => {
     expect(result.current.classProps).toBe('UNSTABLE_PartnerLogo');
   });
 
-  describe('hasSafeAreaDisabled prop', () => {
-    const props = {
-      hasSafeAreaDisabled: true,
-    };
-    const { result } = renderHook(() => usePartnerLogoStyleProps(props));
-
-    it('should return hasSafeAreaDisabled class names', () => {
-      expect(result.current.classProps).toBe('UNSTABLE_PartnerLogo UNSTABLE_PartnerLogo--hasSafeAreaDisabled');
-    });
-
-    it('should not return hasSafeAreaDisabled prop', () => {
-      expect(result.current.props).toStrictEqual({});
+  describe('hasSafeArea prop', () => {
+    const testCases = [
+      { hasSafeArea: false, expectedClassName: 'UNSTABLE_PartnerLogo' },
+      {
+        hasSafeArea: true,
+        expectedClassName: 'UNSTABLE_PartnerLogo UNSTABLE_PartnerLogo--safeArea',
+      },
+    ];
+
+    testCases.forEach(({ hasSafeArea, expectedClassName }) => {
```

Comment:
describe.each(testCases)(`when hasSafeArea is $hasSafeArea`, ({ hasSafeArea, expectedClassName, description }) => {
      const { result } = renderHook(() => usePartnerLogoStyleProps({ hasSafeArea }));

      it(`${description} return safe area class name`, () => {
        expect(result.current.classProps).toBe(expectedClassName);
      });

      it('should not return hasSafeArea prop', () => {
        expect(result.current.props).toStrictEqual({});
      });
    });
  });